docs/plugins.md

```diff
@@ -0,0 +1,33 @@
+# Plugins
+
+Since 0.4.11 go-ipfs has a plugin system for various added functionality.
+
+When IPFS node is created, it will load plugins from `.ipfs/plugins` directory.
```

Comment:
To be precise, this should probably say "will load plugins from `$IPFS_PATH/plugins` (where `$IPFS_PATH` defaults to `~/.ipfs`)" or something like that.